flink-streaming-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/JsonRowSerializationSchemaTest.java

```diff
@@ -35,19 +35,29 @@ public void testRowSerialization() throws IOException {
 		row.setField(1, true);
 		row.setField(2, "str");
 
+		Row resultRow = serializeAndDeserialize(fieldNames, fieldTypes, row);
+		assertEqualRows(row, resultRow);
+	}
 
-		JsonRowSerializationSchema serializationSchema = new JsonRowSerializationSchema(fieldNames);
-		JsonRowDeserializationSchema deserializationSchema = new JsonRowDeserializationSchema(fieldNames, fieldTypes);
+	@Test
+	public void testSerializationOfTwoRows() throws IOException {
```

Comment:
Actually, the test I wanted to propose should serialize two rows (with same schema) using the same `JsonRowSerializationSchema` and `JsonRowDeserializationSchema` to make sure that there are no side effects. This is also what happens in the table source.